core/server/config/env/config.production.json

```diff
@@ -8,5 +8,10 @@
     },
     "paths": {
         "contentPath": "content/"
+    },
+    "logging": {
+        "level": "info",
+        "rotation": true,
+        "transports": ["file"]
```

Comment:
It is possible to register multiple streams with bunyan that operate on different log levels or even different serializers. If we expose this functionality in the config we could have different log files (or even different systems) with different visibility.

Separate logs for errors and requests are quite common in HTTP servers (Apache, Nginx) to get a quick overview of what is going wrong when a request fails. On a high traffic site it's way easier to find what you are looking for when there is a separate error log.

I think that we can add that functionality at a later alpha but it would be very handy to have it available!